frontend/src/assets/locales/ro/translation.json

```diff
@@ -13,5 +13,17 @@
       "dashboard": "Dashboard"
     },
     "collapse": "Restrange meniul"
+  },
+  "organization_profile": {
```

Comment:
this should be called "organization", not necessary to be called "organization_profile".